src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -670,6 +670,7 @@ def aks_create(
     node_count=3,
     nodepool_tags=None,
     nodepool_labels=None,
+    nodepool_taints=None,
```

Comment:
It seems the value is not set to the model, don't we need some kind set_up or update function? Or maybe you could refer to another option `--nodepool-labels`, https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/agentpool_decorator.py#L821-L824